### ISSUE TITLE
Problems with the tutorial

### ISSUE THREAD

**Issue by tvkent** [mojaveazure/angsd-wrapper#12]
Hi,
Here's the list of problems I encountered with the tutorial (though I never even got that far, as you will see):
1. Minor point, but Zea mays ssp. mexicana is listed as Zea mexicana--could be wrong but pretty sure this is a ssp.
2. I believe git comes installed on OSX, does it not? If so, there's no need to install it in the tutorial.
3. `brew install samtools` comes up empty, but it suggests an alternative.  This should be fixed so no one has issues installing. Maybe link to the samtools setup if users have problems using homebrew here?
4. May just be a personal preference, but the command `setup please` is a bit off-putting/patronizing? Seems out-of-convention to use something other than `setup`, anyway.
5. Command name aside, the setup failed when I tried it on a lab computer.  Nothing was setup. I can try it again and give you the output, but you should try your setup command on a few computers that you haven't used for angsd-wrapper yet and see if you can get it to work.

**Comment by rossibarra** [comment]
Agree we should change the "please".

**Comment by mojaveazure** [comment]
1. Fixed
2. I think so, I don't know.
3. I can check this next time I'm at my Mac (buses aren't running on the weekend until school starts). If @pmorrell or @TomJKono could look at this, that'd be awesome.
4. Fixed: it's now `setup dependencies` and `setup data`, depending on what you want to download and setup.
5. I've gotten it running on every computer that I've used, and just tried it on a new computer. I can test more tomorrow. I made a few changes to the `setup` routine, so it may work better for you know.
